qirlib/src/evaluation/jit.rs

```diff
@@ -254,6 +257,23 @@ mod tests {
         Ok(())
     }
 
+    #[serial]
+    #[test]
+    fn fails_if_unknown_external_func() -> Result<(), String> {
```

Comment:
If would be good to see a test in Python to verify that the error comes through correctly.

Reply:
Done in f65e76f.